e2e/rbd_helper.go

```diff
@@ -139,6 +139,10 @@ func createRBDSecret(c kubernetes.Interface, f *framework.Framework) error {
 	}
 
 	err = updateSecretForEncryption(c)
+	if err != nil {
+		return err
+	}
+	err = updateVaultTokenForEncryption(c)
```

Comment:
Because Vault Authentication support can be enabled/disabled, both ways need to be tested. With this change, we are not testing the Kubernetes Auth based authentication anymore?